replicateexistingcluster/replicateexistingcluster.go

```diff
@@ -0,0 +1,185 @@
+package replicateexistingcluster
+
+import (
+	"context"
+	"encoding/json"
+
+	"golang.org/x/xerrors"
+	corev1 "k8s.io/api/core/v1"
+	schedulingv1 "k8s.io/api/scheduling/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	v1 "k8s.io/client-go/applyconfigurations/core/v1"
+	schedulingcfgv1 "k8s.io/client-go/applyconfigurations/scheduling/v1"
+	cfgstoragev1 "k8s.io/client-go/applyconfigurations/storage/v1"
+
+	"github.com/kubernetes-sigs/kube-scheduler-simulator/export"
+)
+
+// Service manages two exportServices.
+// existingClusterExportService is used to reference and export some resources from existing cluster that is indicated by KUBECONFIG.
+// simulatorExportService is used to import(replicate) the resources to the simulator.
+type Service struct {
+	existingClusterExportService *export.Service
+	simulatorExportService       *export.Service
+}
+
+// NewReplicateExistingClusterService initializes Service.
+func NewReplicateExistingClusterService(exportService *export.Service, existingClusterExportService *export.Service) *Service {
+	return &Service{
+		existingClusterExportService: existingClusterExportService,
+		simulatorExportService:       exportService,
+	}
+}
+
+// ImportFromExistingCluster get resources from existing cluster via existingClusterExportService
+// and then apply those to the simulator without restarting it via simulatorExportService.
+func (s *Service) ImportFromExistingCluster() error {
+	ctx := context.Background()
+	expRes, err := s.existingClusterExportService.Export(ctx)
+	if err != nil {
+		return xerrors.Errorf("call Export of existingClusterExportService: %w", err)
+	}
+	impRes, err := cnvtToResourcesForImportFromResourcesForExport(expRes)
+	if err != nil {
+		return xerrors.Errorf("call cnvtToResourcesForImportFromResourcesForExport: %w", err)
+	}
+	// Import to the simulator without restarting it.
+	if err := s.simulatorExportService.Import(ctx, impRes, s.simulatorExportService.IgnoreErr(), s.simulatorExportService.IgnoreRestart()); err != nil {
```

Comment:
Call the `Import` function of the `ExportService` to the simulator.
And it will not restart the simulator.